samples/nrf9160/gnss/README.rst

```diff
@@ -124,7 +144,7 @@ After programming the sample and all the prerequisites to the development kit, y
 2. Open a terminal emulator.
 #. Test the sample by performing the following steps:
 
-   If the default mode is enabled:
+   If NMEA-only mode is not enabled:
```

Comment:
Somehow it would seem more intuitive to first introduce how the NMEA-only mode works rather than explaining how the thing works when NMEA-only mode is not enabled.

Reply:
I was wondering the same, but let it through. I fully agree with Tuomas. Start with the positive.

Reply:
The reason why I changed this was to avoid confusion with the operation modes. In the beginning there weren't different operation modes and the "NMEA-only" was the only "mode" in the sample, so "default mode" couldn't be confused with anything else.

I could change the terms so, that there are "operation" and "output" modes, then the text here could say "the default output mode". How does that sound, @hiltunent and @peknis?

Reply:
I have updated the documentation.